fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalRowCountCache.java

```diff
@@ -113,6 +113,7 @@ public long getCachedRowCount(long catalogId, long dbId, long tableId) {
             if (f.isDone()) {
                 return f.get().orElse(-1L);
             }
+            LOG.info("Row count for table {}.{}.{} is still processing.", catalogId, dbId, tableId);
```

Comment:
is this the main execution path? log.info may print too much to kill the performance.

Reply:
This log will only be printed when the cache is not loaded. Will not print any more after the cache is available.